hive-runner-ios.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
```

Comment:
Not sure why I haven't picked up on this before. There should be a runtime dependency on hive-runner specifying a minimum version. I think:
```ruby
s.add_runtime_dependency 'hive-runner', '~> 2.0.5'
```
should be about right